<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>481</width>
      <height>570</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="img">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>561</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">gridline-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>이미지를 불러와주세요</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/percent.PNG</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_state">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>14</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>state</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>171</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>nemo.PNG</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>52</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>11</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>smile :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>52</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>sad :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_smile">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_sad">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>52</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>angry :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_angry">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>533</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>현재 신뢰도:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_reliability">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>533</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>171</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/menu.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/back.PNG</normaloff>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/back.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>90</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_record">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/record.PNG</normaloff>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/record.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>90</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_backup">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/file.PNG</normaloff>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/file.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>90</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_test">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/test.PNG</normaloff>../../OneDrive/Desktop/AI/프로젝트/얼굴인식프로그램/test.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>90</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
